src/io/flutter/preview/DartElementPresentationUtil.java

```diff
@@ -60,7 +62,7 @@ public static Icon getIcon(@NotNull Element element) {
         if (element.isTopLevelOrStatic()) return STATIC_FIELD_ICON;
         return Field;
       case ElementKind.FUNCTION:
-        return element.isTopLevelOrStatic() ? TOP_LEVEL_FUNCTION_ICON : Function;
+        return element.isTopLevelOrStatic() ? TOP_LEVEL_FUNCTION_ICON : Lambda;
```

Comment:
Shouldn't this also check if the function has a name before using `Lambda`?

Reply:
I'm not sure here - the 'lamda' icon is the one that the Dart plugin is using for non-top level functions, whether the item is actually a lamda or not.

Reply:
Note that this PR is just updating our code to the latest from the Dart plugin - if we think we should change something here we should adjust it upstream.